src/azure-cli/azure/cli/command_modules/serviceconnector/_params.py

```diff
@@ -192,6 +192,11 @@ def add_secret_store_argument(context):
         '--vault-id'], help='The id of key vault to store secret value')
 
 
+def add_configuration_store_argument(context):
```

Comment:
only support resource id? what about also support app config name in same resource group?

Reply:
Using source resource group as default is not supported. We may support this in the future.